### PR TITLE
feat(tailwind): rewrite tailwind component

### PR DESCRIPTION
**Pull request by lordelogos**
## Component / Package Name:
Tailwind

This PR contains:
- [x] bugfix
- [x] feature
- [x] refactor
- [ ] documentation
- [ ] other

Are tests included?

- [ ] yes (_bugfixes and features will not be merged without tests_)
- [x] no

Breaking Changes?

- [x] yes (_breaking changes will not be merged unless absolutely necessary_)
- [ ] no

If yes, please include "BREAKING CHANGES:" in the first commit message body, followed by a description of what is breaking.

### Description
- Re-write tailwind component

### CODE REVIEW

**Comment by shellscape** [comment]
Looks good. Few linting errors and should be able to merge

File: packages/tailwind/src/tailwind.tsx [hunk 1]
```diff
@@ -1,34 +1,115 @@
...
+    // eslint-disable-next-line no-param-reassign
+    element = React.cloneElement(element, element.props, ...processedChildren);
+  }
+
+  return element;
+}
 
-  const fullHTML = renderToStaticMarkup(<>{newChildren}</>);
+function processHead(child: React.ReactElement, responsiveStyles: string[]): React.ReactElement {
+  // FIXME: find a cleaner solution for child as any
```
Comment (lordelogos) [inline packages/tailwind/src/tailwind.tsx:72]: @shellscape do you know a cleaner solution for this check?

Comment (shellscape) [reply packages/tailwind/src/tailwind.tsx:72]: maybe https://stackoverflow.com/a/65794258/416845

Comment (lordelogos) [reply packages/tailwind/src/tailwind.tsx:72]: This doesn't look cleaner, as unknown is just a longer as any

**Pull request merged**